divide2Integers.py

```diff
@@ -0,0 +1,27 @@
+# Time:O(logn)
+# Space:O(1)
+# Leetcode:Yes
+# Issues:None
+
+class Solution:
+    def divide(self, dividend: int, divisor: int) -> int:
+        if dividend == -2147483648 and divisor == -1:

```

Comment:
Pls declare the Integer Min or Integer Max in such places than hard coding the values